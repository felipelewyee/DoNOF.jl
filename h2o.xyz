3

O 0.0 0.0 0.116
H 0.0 0.749 -0.453
H 0.0 -0.749 -0.453
