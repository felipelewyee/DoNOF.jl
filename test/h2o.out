Computing Integrals
Elapsed time:    1.62 Seconds

Number of basis functions                   (NBF)    = 24
Number of auxiliary basis functions         (NBFAUX) = 116
Inactive Doubly occupied orbitals up to     (NO1)    = 0
No. considered Strongly Doubly occupied MOs (NDOC)   = 5
No. considered Strongly Singly occupied MOs (NSOC)   = 0
NO. of Weakly occ. per St. Doubly occ.  MOs (NCWO)   = 3
Dimension of the Nat. Orb. subspace         (NBF5)   = 20
No. of electrons                                     = 10
    No. of alpha electrons                           = 5
    No. of beta electrons                            = 5
Multiplicity                                         = 1

Geometry
========
 O       0.0000000       0.0000000       0.1210000
 H       0.0000000       0.7510000      -0.4850000
 H       0.0000000      -0.7510000      -0.4850000

No violations of the orthonormality
No violations of the orthonormality
 
PNOF8/ADABelief Calculation
==================
 
  Nitext  Nit_orb  Time   Nit_occ  Time         Eelec          Etot           Ediff      maxdiff    MaxGrad_orb  Grad_occ
     1      10    8.8e+00    7    1.1e+01   -83.42261097   -74.29644850 -10082.42261097   0.386956   1.5e+00   1.8e-04 0.00
     2      10    2.1e-03   22    1.3e-03   -84.90546338   -75.77930091    -1.48285241   0.168693   6.7e-01   1.2e-04 0.02
     3      10    2.0e-02   16    8.1e-04   -85.30918968   -76.18302721    -0.40372630   0.059062   2.4e-01   1.4e-04 0.02
     4      10    2.1e-03   10    8.1e-04   -85.35706236   -76.23089989    -0.04787268   0.021596   8.6e-02   1.4e-04 0.02
     5      10    2.8e-03    6    5.8e-03   -85.36347479   -76.23731231    -0.00641242   0.017473   7.0e-02   2.1e-04 0.02
     6      10    1.9e-03    5    3.5e-04   -85.36615695   -76.23999447    -0.00268216   0.017418   7.0e-02   1.6e-04 0.03
     7      10    1.9e-03    3    3.3e-04   -85.36690595   -76.24074348    -0.00074901   0.019969   8.0e-02   1.3e-04 0.03
     8      10    2.0e-03    6    3.9e-04   -85.36727389   -76.24111141    -0.00036793   0.018516   7.4e-02   4.3e-05 0.03
     9      10    3.4e-03    3    3.0e-04   -85.36736711   -76.24120464    -0.00009323   0.019913   8.0e-02   1.6e-04 0.03
    10      10    1.8e-03    4    3.2e-04   -85.36746328   -76.24130080    -0.00009616   0.019005   7.6e-02   1.6e-04 0.03
    11      10    3.3e-03    1    2.2e-04   -85.36759988   -76.24143740    -0.00013660   0.018902   7.6e-02   1.7e-04 0.03
    12      10    3.2e-03    4    3.2e-04   -85.36763828   -76.24147581    -0.00003840   0.017814   7.1e-02   1.4e-04 0.03
    13      10    1.8e-03    1    2.2e-04   -85.36769891   -76.24153643    -0.00006062   0.016864   6.7e-02   1.4e-04 0.03
    14      10    3.4e-03    0    1.8e-04   -85.36769954   -76.24153707    -0.00000063   0.016864   6.7e-02   1.4e-04 0.03
    15      16    4.2e-03    3    3.0e-04   -85.36825824   -76.24209577    -0.00055870   0.000450   1.8e-03   1.7e-04 0.03
 
RESULTS OF THE OCCUPATION OPTIMIZATION
========================================
   1    1.9999717  -20.57448481
   2    1.9870030  -1.25307902
   3    1.9722073  -0.72273754
   4    1.9719947  -0.72158433
   5    1.9793003  -0.52837522
   6    0.0140825  -0.01798608
   7    0.0236312  -0.03250252
   8    0.0236589  -0.03253348
   9    0.0065464  -0.01280893
  10    0.0000127  -0.00025217
  11    0.0056277  -0.00930907
  12    0.0036858  -0.00735943
  13    0.0036557  -0.00737196
  14    0.0033226  -0.00816723
  15    0.0000096  -0.00020136
  16    0.0009895  -0.00182705
  17    0.0006883  -0.00154321
  18    0.0004781  -0.00110599
  19    0.0031280  -0.00789058
  20    0.0000060  -0.00014372
 
----------------
 Final Energies 
----------------
Final NOF Total Energy =     -76.2420958
 
Total Orbital Internal Iterations = 156
Total Occupation Internal Iterations = 91
 
Elapsed time:   26.42 Seconds

---------------------------------
   Lowdin Population Analysis    
---------------------------------
 Idx  Atom   Population   Charge 
---------------------------------
  1     O     8.463      -0.463
  2     H     0.768       0.232
  3     H     0.769       0.231

---------------------------------
   M Diagnostic: 0.03
---------------------------------

